src/utils/sortedBy.js

```diff
@@ -0,0 +1,17 @@
+const sortedBy = (list, typeSorted) => {
+    let sortedList = list;
+    if (typeSorted === "date") {
+        sortedList = list.sort( (a, b) => {
+            let aDate = new Date(a.releaseDate + '/01/01')
```

Comment:
I guess that it will be better to use whole date in your mock data, not only year

Reply:
Resolved